mentorship ios/Models/RelationModel.swift

```diff
@@ -7,18 +7,49 @@
 class RelationModel {
     
     // MARK: - Variables
-    let currentRelation = HomeModel.HomeResponseData.RequestStructure(id: 0, actionUserID: 0, mentor: nil, mentee: nil, acceptDate: 0, startDate: 0, endDate: 0, notes: "")
+    let currentRelation = RequestStructure(id: 0, actionUserID: 0, mentor: nil, mentee: nil, acceptDate: 0, startDate: 0, endDate: 0, notes: "")
     
     let tasks = [TaskStructure]()
     
     let task = TaskStructure(id: 0, description: "", isDone: false, createdAt: 0, completedAt: 0)
     
     // MARK: - Structures
-    struct ResponseData: Decodable {
+    struct ResponseData {
         let message: String?
+        let success: Bool
     }
     
     struct AddTaskData: Encodable {
         var description: String
     }
 }
+
+// MARK: API
+
+extension RequestStructure {
+    func mapTo(viewModel: RelationViewModel) {
```

Comment:
rename to updateViewModel

Reply:
Should I simply rename the function or shift the whole function to the view model (as suggested in the final comment) and change the logic accordingly?